Updated MC parts and minor suggestions (#842)
Materials-Consortia/optimade-python-tools: paper/paper.md

Such esoteric and specialized APIs incur maintenance and usability costs upon both the data providers and consumers, who may not be software specialists.

The OPTIMADE API specification [@andersen2021optimade; @OPTIMADE_spec], released in July 2020, aimed to reduce these costs by designing a common API for use across a consortium of collaborating materials databases and beyond.
Whilst based on the robust JSON:API standard [@JSONAPI], the OPTIMADE API specification presents several domain-specific features and requirements that can be tricky to implement for non-specialist teams.
The repository presented here, `optimade-python-tools`, provides a modular reference server implementation and a set of associated tools to accelerate the development process for data providers, toolmakers and end-users.

# Statement of need

In order to accommodate existing materials database APIs, the OPTIMADE API specification allows for flexibility in the specific data served, but enforces a simple yet domain-specific filter language on well-defined resources.

> This flexibility could be daunting to database providers and maintainers, likely acting to increase the barrier and willingness for hosting an OPTIMADE REST API.

Reviewer: I think I'd undo this bit, this sentence needs to reflect that the flexibility can be a blessing or a curse

Reviewer: ```suggestion
However, this flexibility could be daunting to database providers, likely acting to increase the barrier to hosting an OPTIMADE REST API.
```

- The Materials Project [@MaterialsProject] uses `optimade-python-tools` alongside their existing API [@MAPI] and MongoDB database, providing access to highly-curated density-functional theory calculations across all known inorganic materials.
`optimade-python-tools` handles filter parsing, database query generation and response validation by running the reference server implementation with minimal configuration.
- The NoMaD Repository and Archive [@nomad] integrates the `optimade-python-tools` server in an existing web app and uses the Elasticsearch implementation of the filtering module to allow access to millions of published first-principles calculations submitted by users.
- Materials Cloud [@MaterialsCloud] uses `optimade-python-tools` as a library to provide an OPTIMADE API entry to archived computational materials studies, created with the AiiDA [@AiiDA] Python framework and published through their archive.
In this case, each individual study and archive entry has its own database and separate API entry.
The Python classes within the `optimade` package have been extended to make use of AiiDA and its underlying PostgreSQL [@PostgreSQL] storage engine.

> - ODBX uses `optimade-python-tools` ...

Reviewer: ```suggestion
```

I'd rather not, in case this db disappears soon
